### ISSUE TITLE
Load testing strategy

### ISSUE THREAD

**Issue by koko236** [sockjs/sockjs-client#507]
Hi, 

I wrote a socket server using sockjs. It works fine on development environment. Now I would like to make some load tests, eg. to test the limit of concurrent connections on production server. As far as I know I can't use browser script because of max concurrent connections limitation, and the sockjs-client seems to require browser to run (there are some window.document references). I tried to run it from node.js but it returns with code 2000 - no transports found. 

So my question is what is the best strategy to simulate large number od sockjs connections messages? Do you have any advice or examples (eg. a script, Artillery setup) to run multiple clients simultaneously? If I come up with something I can post it here but any tips will be very appreciated.

**Comment by brycekahle** [comment]
The client supports running from Node.js. What version are you running and how are you using it?

**Comment by koko236** [comment]
Hi, I was just about to write. I have apparently messed something with my setup, when I fixed it it indeed does seem to work! Thanks for the quick response and sorry for the false alarm. Just another silly question then if I already asked one. ;-)
 
SockJS seems to work well out of the box but where can one find more detailed and current documentation for the library? Should I just follow the WebSockets API docs? Apart from readme I found only some links like this one: https://www.rubydoc.info/gems/sockjs#The_Client-Side_Part which document other server implementations and seem to be quite old. I browsed the client source code and it isn't commented very much neither.
